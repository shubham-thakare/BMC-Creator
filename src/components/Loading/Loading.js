import React from "react";
import { Spinner } from "@blueprintjs/core";
import { StyledLoading } from "./styles/Loading.style";

const Loading = () => {
  return (
    <StyledLoading>
      <Spinner />
    </StyledLoading>
  );
};

export default Loading;
