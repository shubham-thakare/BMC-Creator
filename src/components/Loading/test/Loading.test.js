import React from 'react';
import { shallow } from 'enzyme';
import Loading from '..';
import { Spinner } from '@blueprintjs/core';

let wrapper;

describe('Loading Component', () => {
  beforeAll(() => {
    wrapper = shallow(<Loading />);
  });

  it('To match snapshot of Loading', async (done) => {
    expect(wrapper).toMatchSnapshot();
    done();
  });

  it('Spinner exists', async (done) => {
    expect(wrapper.find(Spinner)).toHaveLength(1);
    done();
  });
});
