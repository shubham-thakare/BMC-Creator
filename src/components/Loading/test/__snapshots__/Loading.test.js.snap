// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component To match snapshot of Loading 1`] = `
<Loadingstyle__LoaderWrapper>
  <Blueprint3.Spinner />
</Loadingstyle__LoaderWrapper>
`;
